recipes/elfutils/all/conanfile.py

```diff
@@ -33,111 +38,151 @@ class ElfutilsConan(ConanFile):
         "with_sqlite3": False,
     }
 
-    generators = "pkg_config"
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
 
-    _autotools = None
-    _source_subfolder = "source_subfolder"
+    def disable_werror_flags(self):
+        # This function patches `eu.am` so that warnings are not treated as errors
+        # otherwise the library cannot be built due to warnings related to dereferencing pointers
+        path = os.path.join(self.source_folder, "config", "eu.am")
+        find = '$(if $($(*F)_no_Werror),,-Werror)'
+        repl = '$(if $($(*F)_no_Werror),,)'
+        replace_in_file(self, path, find, repl)
+
+    def enable_static_elfutils(self):
+        # This function patches `configure.ac` to allow building elfutils as a static library
```

Comment:
Could you please keep it as a patch, even if we need to provide a new patch for each version. Plus, it would be great if was possible to send them to the upstream.